modules/osdk-csv-manual-annotations.adoc

```diff
@@ -24,14 +24,18 @@ The following table lists Operator metadata annotations that can be manually def
 |Set a suggested namespace where the Operator should be deployed.
 
 |`operators.openshift.io/infrastructure-features`
-|Infrastructure features supported by the Operator. Users can view and filter by these features when discovering Operators through OperatorHub in the web console. Valid values:
+|Infrastructure features supported by the Operator. Users can view and filter by these features when discovering Operators through OperatorHub in the web console. Valid, case-sensitive values:
 
-- `disconnected`: Operator supports running in disconnected mode. All related images required for mirroring are listed by the Operator.
-- `proxy`: Operator supports running on a cluster behind a proxy. Required environment variables are passed down to Operands.
-- `fipsmode`: Operator supports running in FIPS mode.
+- `disconnected`: Operator supports being mirrored into disconnected catalogs, including all dependencies, and requires no Internet access. All related images required for mirroring are listed by the Operator.
+- `cnf`: Operator provides a Cloud-native Network Functions (CNF) Kubernetes plug-in.
```

Comment:
Using the name that matches https://github.com/cncf/cnf-testbed and format as exists elsewhere in the openshift-docs repo.